### PR TITLE
DOC: Centralize documentation.md

### PR DESCRIPTION
**Pull request by ke-zhang-rd**
This is motivated by https://github.com/bluesky/suitcase-core/issues/76.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,11 @@ This is a package of utility functions and test fixtures for suitcase packages.
...
 pip install suitcase-utils
 ```
+
+## Quick Start
+
+TODO
+
+## Documentation
+
+See the [suitcase documentation](https://nsls-ii.github.io/suitcase).
```
Comment (MikeHart85) [inline README.md:17]: As Maksim pointed out on the suitcase-csv version, it would be better to use https://blueskyproject.io/suitcase/

Comment (ke-zhang-rd) [reply README.md]: thanks

**Comment by ke-zhang-rd** [comment]
> Looks good!

Do you want to merge it?

File: README.md [hunk 2]
```diff
@@ -7,3 +7,11 @@ This is a package of utility functions and test fixtures for suitcase packages.
 ```
 pip install suitcase-utils
 ```
+
+## Quick Start
+
+TODO
```
Comment (mrakitin) [inline README.md:13]: I don't think we should leave it here.

**Pull request merged**